plugins/islandora_context_reaction_restrict_by_ip.inc

```diff
@@ -111,7 +111,7 @@ class IslandoraContextReactionRestrictByIp extends context_reaction {
       if (!strlen($range)) {
         continue;
       }
-      list($low, $high) = explode(':', $range);
+      list($low, $high) = array_pad(explode(':', $range, 2), 2, NULL);
```

Comment:
@MarcusBarnes Personally, I think single IP addresses are clearer, provided we update the help text for that field to to something like

> Enter the IP address ranges, one per line, that are allowed to view objects. Separate the low and high ends of each range with a colon, e.g. 111.111.111.111:222.222.222.222. Asterisks are not allowed. Single IP addresses are also allowed, each on its own line.

But it doesn't really matter to me, as long as users know what to do if they want to allow access from single IPs.